blocks/to-do/to-do.js

```diff
@@ -44,14 +64,37 @@ export default function decorate(block) {
   input.setAttribute('id', 'to-do__input');
   orderedList.classList.add('to-do__ordered-list');
 
+  // Handle if title doesn't exist in the content source
+  if (titleDiv) {
+    titleDiv.classList.add('to-do__title');
+  }
+
+  // Handle error if label doesn't exist
+  if (labelDiv) {
+    labelDiv.replaceWith(labelElement);
+    labelElement.textContent = labelDiv.textContent;
+  }
+
   // Change content
   toDoInputAndBtn.appendChild(input);
   block.appendChild(orderedList);
-  labelDiv.replaceWith(labelElement);
-  labelElement.textContent = labelDiv.textContent;
+
+  // Call getListItems function to fetch data
+  getListItems().then((data) => {
+    // Loop through the data and add list items
+    data.forEach((item) => {
+      // Check if the "Status" is "true" before adding the item
+      if (item.Status === 'true') {
```

Comment:
What is `status === 'true'` , what does it represent? Why don't we add status=false things?